openapi/node.go

```diff
@@ -64,7 +64,8 @@ func (n *node) GetOutputFile() string {
 }
 
 func (n *node) getRef() string {
-	return n.ref
+	panic("(n *node) getRef(): We don't call this one")
```

Comment:
please keep this message consistent with the ones in setChild and SetMetadata. I like the `(n *node) getRef():` part, but `not implemented by composed type` is more clear than `We don't call this one`.

Consider a constant message at the top of the file to return in these methods.

Reply:
sorted. Adjusted the others to match.